api/src/main/java/edu/cornell/mannlib/vitro/webapp/servlet/setup/FileGraphSetup.java

```diff
@@ -121,8 +121,8 @@ public void contextInitialized(ServletContextEvent sce) {
 	private Set<Path> getFilegraphPaths(ServletContext ctx, String... strings) {
 		StartupStatus ss = StartupStatus.getBean(ctx);
 
-		String homeDirProperty = ApplicationUtils.instance().getHomeDirectory().getPath().toString();
-		Path filegraphDir = Paths.get(homeDirProperty, strings);
+		String homeDirPath = VitroHomeDirectory.getHomeSrcPath(ctx);
```

Comment:
I think if we include logic for excludedHomeFiles in the class VitroHomeDirectory, it should be reflected here in the calculation of home directory for rdf files. The same comment is for the RDFFilesLoader class.